api/session_token_login_provider.go

```diff
@@ -30,21 +32,21 @@ var (
 // authenticates the entity with the session token.
 func NewSessionTokenLoginProvider(
 	token string,
-	printOutputFunc func(string, ...any) error,
+	cmdOut io.Writer,
 	updateAccountDetailsFunc func(string) error,
 ) *sessionTokenLoginProvider {
 	return &sessionTokenLoginProvider{
 		sessionToken:             token,
-		printOutputFunc:          printOutputFunc,
+		cmdOut:                   cmdOut,
```

Comment:
I'm not a fan of this naming scheme. It has nothing to do with a cmd, just call it either `output` or `writer`.

Reply:
Renamed to `output`.